staff/lluis-bagur/CV/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
+    <title>CVlluisbaguur</title>
+    
+</head>
+<body>
+<main>
+    <header>
+      <h1>CV Lluis Bagur</h1>
+      <nav style="">
+    <ul>
+        <button><a href="#education" title="index.html">Historial academico</a></button>
```

Comment:
instead of buttons, inside a list (ul), you may use list items (li)

Reply:
I had done it in a previous version, but I thought it looked better on buttons. i will change!